### ISSUE TITLE
Alternative display option for uncommitted changes in the graph

### ISSUE THREAD

**Issue by mhutchie** [mhutchie/vscode-git-graph#393]
**Describe the feature that you'd like**
Provide an alternative display option for uncommitted changes in the graph, so that the open circle is located at the commit referenced by HEAD, instead of at the file system's current state.

![graph-uncommitted-changes-options](https://user-images.githubusercontent.com/5315753/95670491-91632480-0bd7-11eb-97bb-8398b44016b6.png)

**Additional context (optional)**
Raised on behalf of `@acarbonaro` on Discord.

**Comment by mhutchie** [comment]
This will be available in v1.27.0.

If you'd like to use it before the next release, you can download [v1.27.0-beta.3](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.27.0-beta.3), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).

**Comment by adriencarbonaro** [comment]
Great ! Love the display !

**Comment by adriencarbonaro** [comment]
What is the point of having two circles in the right image ?
Other thing, could the current checkout out line display a bold message ?

**Comment by mhutchie** [comment]
Hi @adriencarbonaro,

I understand that for users who choose the "Open Circle at the Checked Out Commit" option, the open circle in the Description column is now somewhat redundant. However, the reason I kept it is that many users work in large repositories with complicated graphs with many simultaneous branch, so the open circle in the Description column is far more noticeable that any symbol in the graph could ever be. If figured there is no harm in keeping it, and it will benefit many users working in large repositories.

I currently bold whichever table row is the current file system's state (i.e. if there are uncommitted changes that will be bolded, otherwise the checked out commit will be bolded). I'm more hesitant to change the bolding for quite a number of reasons based on other user feedback I've received in the past.

I'd like to let this feature great released first, gather feedback from the broader user base, and then consider making further changes (e.g. your bolding suggestion) in a future release once I've gathered more data to make a more informed decision.
